modules/storage-api/src/main/java/org/apache/ignite/internal/storage/index/StorageSortedIndexDescriptor.java

```diff
@@ -150,20 +145,28 @@ private static List<StorageSortedIndexColumnDescriptor> extractIndexColumnsConfi
             CatalogTableDescriptor table,
             CatalogSortedIndexDescriptor index
     ) {
-        assert table.id() == index.tableId() : "tableId=" + table.id() + ", indexTableId=" + index.tableId();
+        assert table.id() == index.tableId() : "indexId=" + index.id() + ", tableId=" + table.id() + ", indexTableId=" + index.tableId();
 
         return index.columns().stream()
                 .map(columnDescriptor -> {
                     String columnName = columnDescriptor.name();
 
                     CatalogTableColumnDescriptor column = table.column(columnName);
 
-                    assert column != null : columnName;
+                    assert column != null : "indexId=" + index.id() + ", columnName=" + columnName;
 
                     CatalogColumnCollation collation = columnDescriptor.collation();
 
                     return new StorageSortedIndexColumnDescriptor(columnName, getNativeType(column), column.nullable(), collation.asc());
                 })
                 .collect(toList());
     }
+
+    private static BinaryTupleSchema createSchema(List<StorageSortedIndexColumnDescriptor> columns) {
```

Comment:
You moved this method to another place. Was there any specific reason for that? It bloats the PR for no good reason

Reply:
For me it looks more convenient that the internal methods are located below, I can return this.

Reply:
fix it